app/src/main/java/org/sopt/jointseminar/melon/presentation/posting/PostingViewModel.kt

```diff
@@ -1,4 +1,35 @@
 package org.sopt.jointseminar.melon.presentation.posting
 
-class PostingViewModel {
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class PostingViewModel : ViewModel() {
+    var review = MutableLiveData<String>()
+    var reviewCount = MutableLiveData<Int>()
+    var canRegister = MutableLiveData<Boolean>()
+    var countString = MutableLiveData<String>()
+
+    init {
+        countString.value="0"
+    }
+
+    fun updateCount(){
+        if(review.value==null)
+            reviewCount.value=0
+        else
+            reviewCount.value = review.value?.length
+    }
+
+    fun checkRegister() : Boolean {
+        if(!review.value.isNullOrBlank()){
+            canRegister.value=true
+            return true
+        }
+        else
+            return false
```

Comment:
```suggestion
        return if (!review.value.isNullOrBlank()) {
            canRegister.value = true
            true
        } else {
            false
        }
```

굳이 변경하지 않으셔도 됩니다 ㅎㅎ 이렇게 작성해볼 수 있을 것 같아요 ~!
그리고 if문에 괄호 쓰면 else문(return문 한줄만 있긴하지만 ^____^*)도 괄호써서 좀 더 정리돼 보이도록 수정해도 좋을 것 같아요~!